Client/core/CSettings.cpp

```diff
@@ -1629,12 +1643,18 @@ void CSettings::UpdateVideoTab()
     float fPos = SharedUtil::Unlerp(g_pCore->GetMinStreamingMemory(), uiStreamingMemory, g_pCore->GetMaxStreamingMemory());
     m_pStreamingMemory->SetScrollPosition(fPos);
 
+    // Radar map alpha
     int iVar = 0;
     CVARS_GET("mapalpha", iVar);
     int iAlphaPercent = ceil(((float)Clamp(0, iVar, 255) / 255) * 100);
     m_pMapAlphaValueLabel->SetText(SString("%i%%", iAlphaPercent).c_str());
     float sbPos = (float)iAlphaPercent / 100.0f;
     m_pMapAlpha->SetScrollPosition(sbPos);
+
+    // Radar map image
+    CVARS_GET("radar_map_image", iVar);
+    if (iVar == 0 || iVar == 1)
```

Comment:
Does CVAR class guarantee that this check is always valid?

Reply:
Cvars get does place the value in the int variable. The value is clamped by the settings script do it's always right.

Reply:
So you can remove this check